sdk/storage/azure-storage-file/HISTORY.md

```diff
@@ -5,6 +5,42 @@
+        - `FileClient`: The client handles operations for a particular file. This includes creating or deleting that file, as well as upload and download data and managing properties.
+
+        These clients can be accessed by navigating down the client hierarchy, or instantiated directly using URLs to the resource (account, share, directory or file).
+      For full details on the new API, please see reference documentation.
+    - The copy file operation now return a polling object that can be used to check the status of the operation, as well as abort the operation.
+    - The `close_handles` operation now return a polling object that can be used to check the status of the operation.
+    - Download operations now return a streaming object that can download data in multiple ways:
+        - Iteration: The streamer is an iterable object that will download and yield the content in chunks. Only supports single threaded download.
+        - `content_as_bytes`: Return the entire file content as bytes. Blocking operation that suppots multi-threaded download.
+        - `content_as_text`: Return the entire file content as decoded text. Blocking operation that supports multi-threaded download.
+        - `download_to_stream`: Download the entire content to an open stream handle (e.g. an open file). Supports multi-threaded download.
+    - New underlying REST pipeline implementation, based on the new `azure.core` library.
+    - Client and pipeline configuration is now available via keyword arguments at both the client level, and per-operation. See reference documentation for a full list of optional configuration arguments.
+    - New error hierarchy:
+        - All service errors will now use the base type: `azure.core.exceptions.HttpResponseError`
+        - The are a couple of specific exception types derived from this base type for common error scenarios:
+            - `ResourceNotFoundError`: The resource (e.g. queue, message) could not be found. Commonly a 404 status code.
+            - `ResourceExistsError`: A resource conflict - commonly caused when attempting to create a resource that already exists.
+            - `ResourceModifiedError`: The resource has been modified (e.g. overwritten) and therefore the current operation is in conflict. Alternatively this may be raised if a condition on the operation is not met.
+            - `ClientAuthenticationError`: Authentication failed.
+    - Operation `set_file_properties` has been renamed to `set_http_headers`.
+    - Operations `get_file_to_<output>` have been replaced with `download_file`. See above for download output options.
+    - Operations `create_file_from_<input>` have been replace with `upload_file`.
+    - Operations `get_share_acl` and `set_share_acl` have been renamed to `get_share_access_policy` and `set_share_access_policy`.
+    - Operation `set_share_properties` has been renamed to `set_share_quota`.
```

Comment:
Should it be `set_share_metadata`  and `set_share_quota` ?

Reply:
No - set_share_metadata is still there, there used to be a set_share_properties operation that is now just set_share_quota